### ISSUE TITLE
Usability: NVDA: Position and Size of list item in 'Create new Workbook' window is not announced by NVDA

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#394]
### Steps to Reproduce

1. Launch Xamrin Workbook application.
2. Navigate to List view item in 'Create new Workbook' and observe NVDA announcement

### Expected Behavior
Position and Size of list item should be announced by NVDA
### Actual Behavior
Position and Size of list item is not announced by NVDA

Narrator behavior: Narrator is announcing the Position and Size of list item as : 'Xamarin.Interactive.Client.ViewControllers.NewWorkbookItem, 4 of 4, selected'
NVDA behavior: NVDA is not announcing the Position and Size of list item as 'Xamarin.Interactive.Client.ViewControllers.NewWorkbookItem'
### User Impact
Users who rely on screen reader will not be able to know the position and size of list items
### Screenshots
![position_list_item_inspect](https://user-images.githubusercontent.com/22956130/40775850-bc47c204-64e6-11e8-8bbd-cdb434835fcd.png)
![position_list_item_narrator](https://user-images.githubusercontent.com/22956130/40775867-c154fc08-64e6-11e8-829b-a3583eaaab12.png)
![position_list_item_nvda](https://user-images.githubusercontent.com/22956130/40775869-c1830ae4-64e6-11e8-8f3a-7e6b64145d40.png)

### Environment
Windows :Widows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yUsable
Accessibility
Xamarin-Workbooks-Windows
Wipro
